docs/_ext/scrapycoverage.py

```diff
@@ -0,0 +1,291 @@
+"""
+    scrapycoverage
+    ~~~~~~~~~~~~~~
+
+    Check Python modules and C API for coverage.
+
+    Fork of sphinx.ext.coverage with the following changes:
```

Comment:
Hey! How hard would it be to either get changes merged to sphinx, or create a separate package for this (we can use scrapy github org)? It looks risky to have such code in `docs/_ext`, with not tests, etc.

Reply:
I’ll try to get it merged upstream, with a few other changes.

Reply:
That'd be perfect!